src/App.js

```diff
@@ -1,11 +1,57 @@
 import "./App.css";
+import Header from "./components/Header.js";
+import ToDoList from "./components/ToDoList.js";
+import { useState } from "react";
+
+// const toDoArray = [
+//   { name: "shoppingItem", status: "Pending" },
+//   { name: "running", status: "Done" },
+// ];
+
+export default function App() {
+  const [toDo, setToDo] = useState([]);
```

Comment:
I'd call the state `toDos` because it's holding an array of objects, each object representing a todo.
Also I'd change the `setToDo` function to be called `setToDos`, to follow the convention.